libosmscout/include/osmscout/DataFile.h

```diff
@@ -144,16 +150,14 @@ namespace osmscout {
   /**
    * Read one data value from the given file offset.
    *
-   * Method is thread-safe.
+   * Method is NOT thread-safe.
    */
   template <class N>
   bool DataFile<N>::ReadData(const TypeConfig& typeConfig,
                              FileScanner& scanner,
                              FileOffset offset,
                              N& data) const
   {
-    std::lock_guard<std::mutex> lock(accessMutex);
```

Comment:
Lukas and Tim: it looks like this PR has made some methods non-thread safe. Can you tell what are the consequences of that?